### PR TITLE
Adding Microsoft.Storage 2017-10-01

### PR DESCRIPTION
**Pull request by JasonYang-MSFT**

### CODE REVIEW

File: tests/2017-10-01/Microsoft.Storage.tests.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
+    "tests": [
```
Comment (TianoMS) [inline tests/2017-10-01/Microsoft.Storage.tests.json]: >     [](start = 0, length = 4)

please format the test file with consistent indents.

**Comment by Tiano2017** [comment]
Looks good. Only one small issue on formatting.

File: tests/2017-10-01/Microsoft.Storage.tests.json (hunk 1 above)
Comment (JasonYang-MSFT) [reply tests/2017-10-01/Microsoft.Storage.tests.json]: Sure, just a minute.

**Comment by JasonYang-MSFT** [comment]
@TianoMS Re-formatted to align the indents with other tests.

**Pull request merged**